src/types.ts

```diff
@@ -0,0 +1,8 @@
+export type FetchEphemeralKeyFunction = (fetchParams: {
```

Comment:
Not sure how to be share or redefine types across packages, defaulted to duplicating for the time being.

Reply:
Will do in a follow-up!